sdk/spring/spring-cloud-azure-autoconfigure/src/test/java/com/azure/spring/cloud/autoconfigure/aad/implementation/webapp/AadIdTokenRolesExtractionTests.java

```diff
@@ -20,7 +21,7 @@ class AadIdTokenRolesExtractionTests {
 
     private AadOAuth2UserService getUserService() {
         AadAuthenticationProperties properties = mock(AadAuthenticationProperties.class);
-        return new AadOAuth2UserService(properties);
+        return new AadOAuth2UserService(properties, new RestTemplate());
```

Comment:
Same here, mock the rest template instead of new.

Reply:
Will update this.